docs/usage/faq.md

```diff
@@ -29,6 +29,11 @@ The Renovate team only fixes bugs in an older version if:
 If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
 If you're self hosting Renovate, use the latest release if possible.
 
+## Why is the Mend hosted app using a older version compared to source?
+
+The Renovate maintainers manually update the hosted app.
+This means the hosted app can lag a few hours to a week behind the open source version.
```

Comment:
@rarkins commented in another PR:

> updates in the hosted app are approved manually, typically in <7 days from once the official image is published to Docker Hub

The `typically in <7 days` part is probably the most important thing users want to know.

Is `once the official image is published to Docker Hub` important for our users? It may be too much implementation-level detail. Maybe we can say: "The maintainers aim to update the hosted app within seven days of releasing a new open source version."

Here's a suggestion for @rarkins to edit: 😉 

```suggestion
The Renovate maintainers manually update the hosted app.
They aim to update the app within seven days of publishing a newer image on Docker Hub.
```

Reply:
I think the existing wording is ok